lib/credit_check.rb

```diff
@@ -14,15 +16,29 @@ def two_times(array)
     end
   end
 
-  def sum_greater_than_ten(array)
-    results = array.map.with_index do |num, index|
+  def sum_digits_over_ten
+    @digits.map!.with_index do |num, index|
       if num > 9
         num - 9
       else
         num
       end
     end
-    @card_number = results.sum
+  end
+
+  def results_summed
+    @digits = @digits.sum
```

Comment:
In this case, replacing @digits feels wrong, because you now have only a sum, not digits.